pkg/cli/testdata/merge_logs_v2/results/3.non-standard

```diff
@@ -0,0 +1,20 @@
+I181130 22:14:34.828612 740 storage/store_rebalancer.go:277  [n1,s1,store-rebalancer] load-based lease transfers successfully brought s1 down to 37128.65 qps (mean=29834.62, upperThreshold=37293.27)
```

Comment:
I'm going to be honest, I don't really understand what's going on here. Isn't this just the vanilla v1 format?

Reply:
The test decodes crdb-v2 formatted log entries and encodes them as crdb-v1. More work needs to be done for encoding crdb-v2 log entries. At the time being, `testdata/merge_logs_v2/results` and `testdata/merge_logs/results` are identical except for `results/5.*` because the corresponding test cases have log entries with no logging tags - the two parsers digest this situation differently. For the crdb-v2 parser,  `-` is stored when there are no logging tags whereas the crdb-v1 parser is fine with the empty string.